### PR TITLE
[GLUTEN-6425][CH] Support day time internval

### PR DESCRIPTION
**Pull request by lgbo-ustc**
## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)

Fixes: #6425

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)

unit tests

(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)

### CODE REVIEW

File: gluten-core/src/main/scala/org/apache/gluten/expression/ExpressionConverter.scala [hunk 1]
```diff
@@ -686,6 +686,21 @@ object ExpressionConverter extends SQLConfHelper with Logging {
         LiteralTransformer(Literal(Math.E))
       case p: Pi =>
         LiteralTransformer(Literal(Math.PI))
+      case dateAdd: DateAdd =>
+        logError(s"xxx handle dateAdd")
```
Comment (zzcclp) [inline gluten-core/src/main/scala/org/apache/gluten/expression/ExpressionConverter.scala:690]: remove ？

**Comment by lgbo-ustc** [comment]
Run Gluten Clickhouse CI

**Comment by lgbo-ustc** [comment]
Run Gluten Velox CI

**Review by zzcclp — APPROVED** [review]
LGTM

**Pull request merged**